modules/composer-service-remote/src/main/java/org/opencastproject/composer/remote/ComposerServiceRemoteImpl.java

```diff
@@ -524,6 +524,35 @@ public Job composite(Dimension compositeTrackSize, Option<LaidOutElement<Track>>
     }
   }
 
+  public Job mergeAudioTracks(String profileId, List<Long> audioStartTimes, List<Track> audioTracks) throws EncoderException, MediaPackageException {
```

Comment:
Note for future re-review: This *must* be tested in a distributed install prior to merge!